hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenAirdropTest.java

```diff
@@ -265,6 +261,121 @@ final Stream<DynamicTest> chargeAssociationFeeForNFT() {
+                                getTxnRecord("first airdrop")
+                                        .hasPriority(recordWith()
+                                                .pendingAirdrops(includingFungiblePendingAirdrop(moving(
+                                                                10, FT_FOR_MULTIPLE_PENDING_TRANSFER)
+                                                        .between(OWNER, RECEIVER_WITHOUT_FREE_AUTO_ASSOCIATIONS)))
+                                                .pendingAirdrops(includingNftPendingAirdrop(movingUnique(
+                                                                NFT_FOR_MULTIPLE_PENDING_TRANSFER, 1L)
+                                                        .between(OWNER, RECEIVER_WITHOUT_FREE_AUTO_ASSOCIATIONS)))),
+                                getTxnRecord("second airdrop")
+                                        .hasPriority(recordWith()
+                                                .pendingAirdrops(includingFungiblePendingAirdrop(moving(
+                                                                20, FT_FOR_MULTIPLE_PENDING_TRANSFER)
+                                                        .between(OWNER, RECEIVER_WITHOUT_FREE_AUTO_ASSOCIATIONS)))
+                                                .pendingAirdrops(includingNftPendingAirdrop(movingUnique(
+                                                                NFT_FOR_MULTIPLE_PENDING_TRANSFER, 2L)
+                                                        .between(OWNER, RECEIVER_WITHOUT_FREE_AUTO_ASSOCIATIONS)))),
+                                // assert account balances
+                                getAccountBalance(RECEIVER_WITHOUT_FREE_AUTO_ASSOCIATIONS)
+                                        .hasTokenBalance(FT_FOR_MULTIPLE_PENDING_TRANSFER, 0)
+                                        .hasTokenBalance(NFT_FOR_MULTIPLE_PENDING_TRANSFER, 0),
+                                getAccountBalance(OWNER)
+                                        .hasTokenBalance(FT_FOR_MULTIPLE_PENDING_TRANSFER, 1000)
+                                        .hasTokenBalance(NFT_FOR_MULTIPLE_PENDING_TRANSFER, 10L),
+                                validateChargedUsd("first airdrop", 0.2, 10),
+                                validateChargedUsd("second airdrop", 0.15, 10));
+            }
+
+            // AIRDROP_21
+            @HapiTest
+            final Stream<DynamicTest>
+                    transferOneFTTwiceFromEOAWithOneFTInBalanceToAccountWithNoFreeAutoAssociationsResultsInPendingAggregated() {
+                var sender = "sender";
+                return defaultHapiSpec(
+                                "Send one FT from EOA with only One FT in balance twice to Account without free Auto-Associations")
+                        .given(
+                                cryptoCreate(sender).maxAutomaticTokenAssociations(-1),
+                                tokenAirdrop(moving(1, FUNGIBLE_TOKEN).between(OWNER, sender))
```

Comment:
you should use cryptoTransfer if you are just sending some balance to the sender, not a big deal but I think its more readable

Reply:
Also you don't need to validate the transaction record as this is not the functionality you are testing here.